src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobRejecter.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.sas.BlobContainerSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.services.storage.RejectedFilesHandler.REJECTED_CONTAINER_SUFFIX;
+
+@Component
+public class BlobRejecter {
```

Comment:
The blob is rejected earlier and this tool just moves it. Maybe `RejectedBlobMover` or something like that?

Reply:
Note that the duplicate handler will use this class as well

Reply:
So `BlobMover`?

Reply:
renamed to `BlobMover`, method is called `moveToRejectedContainer`